plugin/trino-hive/src/main/java/io/trino/plugin/hive/orc/OrcPageSourceFactory.java

```diff
@@ -123,16 +124,23 @@
     private static final Pattern DEFAULT_HIVE_COLUMN_NAME_PATTERN = Pattern.compile("_col\\d+");
     private final OrcReaderOptions orcReaderOptions;
     private final HdfsEnvironment hdfsEnvironment;
```

Comment:
you can remove `hdfsEnvironment` now. It seems unused

Reply:
It is still used to create OrcPageSource

Reply:
It is still used, it is passed to `io.trino.plugin.hive.orc.OrcPageSourceFactory#createOrcPageSource` and then used to create `OrcDeletedRows`.

Reply:
We could try to simplify it further